pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -676,8 +676,6 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 			if err != nil {
 				log.Errorf("Delete PV %s failed with error: %+v", pvName, err)
 			}
-			setInstanceError(ctx, r, instance,
```

Comment:
If PV deletion fails then where do we show error? I m not sure why we are removing this error?

Reply:
Done